.golangci.yml

```diff
@@ -15,8 +15,24 @@ linters-settings:
     # see https://github.com/kisielk/errcheck#excluding-functions for details
     exclude: ./.errcheck-exclude
 
-  goimports:
-    local-prefixes: "github.com/grafana/mimir"
+  gci:
+    # Section configuration to compare against.
+    # Section names are case-insensitive and may contain parameters in ().
+    # The default order of sections is `standard > default > custom > blank > dot`,
+    # If `custom-order` is `true`, it follows the order of `sections` option.
+    # Default: ["standard", "default"]
+    sections:
+      - standard
+      - default
+      - prefix(github.com/grafana/mimir)
+
+    # Skip generated files.
+    # Default: true
+    skip-generated: false
```

Comment:
Why not skip generated files?

Reply:
I'm not sure. I turned these off. I guess we don't want it to lint generated pb.go files